### ISSUE TITLE
Python versions

### ISSUE THREAD

**Issue by airboy182** [wyager/LEDStrip#3]
So when I try installing everything thats required, I get syntex errors, Depending on which version of python and other misc stuff I get different errors. Can you tell me what version of everything you are using?

Thank you in advanced.

**Comment by gusnaughton** [comment]
I can confirm this works on Python 2.7.6.

**Comment by wyager** [comment]
Works on Python 2.7.10 as well, for me.

@airboy182 what's your python setup?

**Comment by airboy182** [comment]
I tried with the latest python, which didn't work on Mac or windows, which
is fair for windows as it wasn't claimed to work on windows, on Linux to
get it working properly, I had to roll back to...I think it was version is
2.6 maybe? To be honest I asked so long ago it forgot about this. And the
computer I got it working on (kinda) has be wiped since then, I need to
reset it back up anyway so I will try again and see.
